### PR TITLE
adds to documentation

### PR DESCRIPTION
**Pull request by agnesgaroux**
## What does this change?

Add step by step instructions on how to run the commands 

## How to test

N/A

## How can we measure success?

Someone with little prior knowledge of the workflow should be able to run the steps

## Have we considered potential risks?

N/A

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,9 +7,16 @@ shoots/clean:
...
+# transfer process separately without overwhelming the target system.
+# The right number for archivematica is probably about 20.
+
+%.sliced: %
+	split -l 20 $< $<.
+
 # Request the Glacier restoration of the shoots in the given file
 # The file is expected to contain one shoot identifier per line.
-# In order to run this, set your AWS profile to one with authority in the workflow account.
+# In order to run this, set your AWS profile to one with authority in the platform account.
```
Comment (agnesgaroux) [inline Makefile:19]: The `[wellcomecollection-editorial-photography](https://eu-west-1.console.aws.amazon.com/s3/buckets/wellcomecollection-editorial-photography?region=eu-west-1)` is in the platform account

File: Makefile [hunk 2]
```diff
@@ -49,4 +49,4 @@ shoots/clean:
 	cat % | python src/touch.py staging
 
 %.touched.production: %
-	cat % | python src/touch.py production
+	cat $< | python src/touch.py production
```
Comment (agnesgaroux) [inline Makefile:52]: Couldn't make it work by feeding the list of S3 key directly through the command line so this now takes a file as input like the other commands

**Review by kenoir — APPROVED** [review]
This looks like a nice improvement on the docs to me, but I have not followed the instructions on my machine _and_ you may want @paul-butcher to review this so a +1 from me but up to you if you want to seek further review.

**Review by paul-butcher — COMMENTED** [review]
Nice!

**Review by paul-butcher — APPROVED** [review]

**Pull request merged**